### ISSUE TITLE
What is the loss function of yolo3? How should the formula be written?

### ISSUE THREAD

**Issue by ghost** [qqwweee/keras-yolo3#388]
What is the loss function of yolo3? How should the formula be written?The formula was not written in the paper.Does anyone know?

**Comment by lcltopismine** [comment]
Please refer to yolo V1 for a detailed loss function.
By the way, the author of this repo changed loss function a little, for part 4. I guess so
